tests/plugins/richcombo/manual/arialabel.md

```diff
@@ -0,0 +1,28 @@
+@bender-ui: collapsed
+@bender-tags: 4493, 4.16.1, bug
+@bender-ckeditor-plugins: wysiwygarea, stylescombo, toolbar
+
+**Note:** This is the test for screen readers, so open one before testing.
+
+1. Press <kbd>Tab</kbd> key multiple times to move focus to the editor.
+2. Press <kbd>Alt+F10</kbd> to switch focus to the toolbar and navigate to the `Styles` combo.
```

Comment:
Since there is only one combo in the toolbar. We don't need additional navigation in this step. Simple `Alt+F10` keys press put us in a right place.

Reply:
That's true for tests run on dev version of the editor, but on built version there will be more buttons. That's why I included more verbose instructions.